src/main/java/com/greenfox/erste/repository/ICardRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.greenfox.erste.repository;
+
+import com.greenfox.erste.Models.Card;
+import org.springframework.data.repository.CrudRepository;
+
+public interface ICardRepository extends CrudRepository<Card, Long>{
```

Comment:
The parameters CrudRepository<Card, Long> should be changed because the Hash field (String) is an id for Card object

Reply:
no longer valid